File: prob9.py
import sys
nums = range(1,997)

for x in nums:
    for y in nums:
        for z in nums:
            if (x+y+z == 1000):
                l = [x,y,z]
                l.sort()
                a,b,c = l
                if a**2 + b**2 == c**2:
                    print(a,b,c)
                    print(a*b*c)
                    sys.exit()
                    
                    

File: prob9.go
package main

import (
	"fmt"
	"os"
	"sort"
)

func main() {
	nums := []int{}
	for i := 1; i < 997; i++ {
		nums = append(nums, i)
	}
	for _, x := range nums {
		for _, y := range nums {
			for _, z := range nums {
				if x+y+z == 1000 {
					l := []int{x, y, z}
					sort.Ints(l)
					a, b, c := l[0], l[1], l[2]
					if a*a+b*b == c*c {
						fmt.Println(a, b, c)
						fmt.Println(a * b * c)
						os.Exit(0)
					}
				}
			}
		}
	}
}
